app/screens/browse_channels/channel_dropdown.tsx

```diff
@@ -73,9 +72,12 @@ export default function ChannelDropdown({
             items += 1;
         }
 
-        showModalOverCurrentContext(Screens.BOTTOM_SHEET, {
+        bottomSheet({
+            title: '',
```

Comment:
title and closeButtonId are important cause of the way bottomSheets are displayed on Tablets. If you want I can take care of this for you